### PR TITLE
[9.0][MIG] sale_validity

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Backport of https://github.com/OCA/sale-workflow/pull/427.

I tried to keep the *Default Validity of Sale Orders* in the configuration of 'Sales', but since the <code>sale.config.settings</code> doesn't have the 'company_id' in v9 I eventually prefer to keep it simple and let it in the company configuration.

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]
Thanks for the contribution

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Technical and code review. Thanks

File: sale_validity/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+=====
+
+When you create a new quotation, the *Expiration Date* will be set by default
+to today's date plus the number of days configured in *Sale Settings* page.
+If you modify the *Order Date* field, the *Expiration Date* will be updated
+accordingly.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/167/10.0
```
Comment (JordiBForgeFlow) [inline sale_validity/README.rst]: Change to 9.0

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional refiew

**Pull request merged**